### PR TITLE
Add todo form added

### PR DESCRIPTION
**Pull request by aniketrathi**
## Description
Created a form to store a Todo item in  database.

### Empty:
Screenshot (44)

### Success: Title Stored in MongoDB
Screenshot (42)

### Error:
Screenshot (43)

### Automated test cases added
- None

### Manual test cases run
- Checked input with a string, worked perfectly.
- Checked MongoDB, todo is stored perfectly on submit the form.
- If title is empty, it shows error "Please enter a valid title".

### Browsers tested
- [x] Chrome
- [x] Safari
- [x] Internet Explorer
- [x] Firefox

## Platform dependencies
- None

### CODE REVIEW

File: src/app/components/app/app.styles.css [hunk 1]
```diff
@@ -6,3 +6,10 @@ ul#menu li {
   display:inline;
   padding: 10px;
 }
+
+.title{
+  font-size: 3vw;
```
Comment (vindru) [inline src/app/components/app/app.styles.css]: Not sure about vw, 'rem' / 'px' is standard

File: src/app/components/app/app.styles.css [hunk 2]
```diff
@@ -6,3 +6,10 @@ ul#menu li {
...
+  font-weight: bold;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  color: indigo;
```
Comment (vindru) [inline src/app/components/app/app.styles.css]: use color codes

File: src/app/components/todo/todo.styles.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.text-error {
+  font-size: 15px;
+  color: red;
+}
+
+#title {
+  border-bottom: 1px solid black;
+}
```
Comment (vindru) [inline src/app/components/todo/todo.styles.css:8]: add eof

Comment (aniketrathi) [reply src/app/components/todo/todo.styles.css:8]: done

File: src/app/components/app/app.styles.css (hunk 2 above)
Comment (aniketrathi) [reply src/app/components/app/app.styles.css]: ok

File: src/app/components/app/app.styles.css (hunk 1 above)
Comment (aniketrathi) [reply src/app/components/app/app.styles.css]: view width that means font-size directly proportional to the width of the screen

**Pull request merged**